### DISCUSSION TITLE
Enable/disable automatic board deletion

### DISCUSSION THREAD

**Discussion by SelinaBuff** [inovex/scrumlr.io#3004] [Proposals]
Delete board after (6 weeks) Function in menu (activate/deactivate).

**Comment by Schwehn42** [comment]
what should the default value be?

**Reply by brandstetterm** [reply]
The default should probably be that it gets deleted after 6 weeks. But I guess the option to disable the automatic deletion should be place **very prominently** somewhere on the board creation screen
